app/controllers/users_controller.rb

```diff
@@ -13,9 +18,45 @@ def create
     end
   end
 
+  # GET /users/1/edit
+  def edit
+    set_user
+  end
```

Comment:
Change the edit method to
`  def edit`
    `@user = User.find(params[:id])`
  `end`

Reply:
Hello, we have the set_user defined already. and it is working fine.